speculos/mcu/seproxyhal.py

```diff
@@ -288,6 +289,24 @@ def can_read(self, s: socket.socket, screen):
 
             if tag == SephTag.GENERAL_STATUS:
                 if int.from_bytes(data[:2], 'big') == SephTag.GENERAL_STATUS_LAST_COMMAND:
+                    if self.refreshed:
+                        self.refreshed = False
+
+                        if not screen.nbgl.disable_tesseract:
+                            # Pause flow of time while the OCR is running
+                            self.time_ticker_thread.pause()
+
+                            # Run the OCR
+                            screen.nbgl.m.update_screenshot()
+                            screen_size, image_data = screen.nbgl.m.take_screenshot()
```

Comment:
Can't we only call `screen.nbgl.m.take_screenshot()` only once between here and in `screen.nbgl.m.publish_screenshot()` ?